src/ingestor/typedData/index.ts

```diff
@@ -103,7 +109,7 @@ export default async function ingestor(body) {
   }
 
   let legacyBody = {
-    address: body.address,
+    address: message.from,
```

Comment:
It won't be a problem if i pass someone's address into `from`? I have to run few tests

Reply:
Run your test

Reply:
Sorry for delay, this is working perfectly as far as i tested 👍